src/extension/commands/run_query_file.ts

```diff
@@ -24,14 +24,15 @@
 import * as vscode from 'vscode';
 import {runMalloyQuery} from './run_query_utils';
 import {WorkerConnection} from '../worker_connection';
+import {ResultJSON} from '@malloydata/malloy';
 
-export function runQueryFileCommand(
+export async function runQueryFileCommand(
   worker: WorkerConnection,
   queryIndex = -1
-): void {
+): Promise<ResultJSON | undefined> {
```

Comment:
Just curious, why is this change needed?

Reply:
VS Code will plumb results back from commands, but we were not returning anything. I use the to validate the results of the query that was run. I updated all the query running commands to return results, although I'm only using the one at the moment.